src/main/java/com/conveyal/datatools/manager/auth/Auth0UserProfile.java

```diff
@@ -376,6 +374,26 @@ public boolean canAdministerProject(String projectID, String organizationId) {
         return false;
     }
 
+    public boolean canAdministerProject(FeedSource feedSource) {
+        return canAdministerProject(feedSource.projectId, feedSource.organizationId());
+    }
+
+    public boolean canAdministerProject(com.conveyal.datatools.manager.models.Project project) {
```

Comment:
Can we do this?

```suggestion
    public boolean canAdministerProject(Project project) {
```

Reply:
I don't think so because of the other Project class defined in this file.